docs/source/operations/besu_networkyaml.md

```diff
@@ -28,7 +28,7 @@ The sections in the sample configuration file are
 
 `type` defines the platform choice like corda/fabric/indy/quorum/besu, here in the example its **besu**.
 
-`version` defines the version of platform being used. The current Hyperledger Besu version support is only for **1.4.4**.
+`version` defines the version of platform being used. The current Hyperledger Besu version support is only for **1.5.5**.
```

Comment:
are we removing support for 1.4.4?

Reply:
yes, since the besu team is not actively developing multiple versions and the network is forward compatible, we are removing the 1.4.4 support. 
![10e6ac61-236d-4799-8dd8-0a78600b1e6e](https://user-images.githubusercontent.com/67091468/102768707-97c80680-43a7-11eb-8dad-0bfa70be188c.jpg)